tpe/plugin/src/SimulationFeatures.cc

```diff
@@ -41,6 +43,8 @@ void SimulationFeatures::WorldForwardStep(
       << _worldID.id
       << "] not found."
       << std::endl;
+    // TODO(adlarkin) return here? de-referencing it below won't work
+    // if it == this->worlds.end()
```

Comment:
I noticed that there's a check to see if the world wasn't found, but we don't seem to stop early if the world isn't found. Should I add a `return` statement here?

Reply:
yeah I think we can return here otherwise it may just crash

Reply:
Added in https://github.com/ignitionrobotics/ign-physics/pull/223/commits/2df9ea0c290f75ea72a9781ba003ca0b84d15ad1